doc/classes/WindowDialog.xml

```diff
@@ -41,7 +41,7 @@
 			The vertical offset of the close button.
 		</theme_item>
 		<theme_item name="panel" type="StyleBox">
-			The style for both the content background of the [WindowDialog] and the title bar.
+      The style for both the content background of the [WindowDialog] and the title bar. The title bar is created with a top border and an expand margin of [code]panel[/code].
```

Comment:
Here's a suggestion I feel makes the description read a little more clearly in plain English. Also, it restores the tabs for indentation, that were changed into spaces.

```suggestion
			The style for both the content background of the [WindowDialog] and the title bar. The title bar is created with a top border and an expand margin using the [code]panel[/code] stylebox.
```